includes/admin/upgrades/upgrade-functions.php

```diff
@@ -142,6 +142,10 @@ function give_do_automatic_upgrades() {
 		case version_compare( $give_version, '2.5.11', '<' ):
 			give_v2511_upgrades();
 			$did_upgrade = true;
+
+		case version_compare( $give_version, '2.6.3', '<' ):
```

Comment:
@mehul0810 this is a pretty massive update to Stripe so we're going to hold off releasing this until `2.7.0` to give adequate time for testing and documentation - please update the version number throughout to reflect that as well as the upgrade process.

Reply:
@DevinWalker Can you also let me know what version you are planning to release along with Give 2.7.0 for Recurring, Stripe Premium, and Fee Recovery?